charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 5.1.3
+version: 5.1.4
```

Comment:
as this is a new feature version should be 5.2.0

Reply:
That's been bumped now